### PR TITLE
Update connecting to testnet instructions.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Update connecting to testnet instructions.

### CODE REVIEW

File: docs/HowTo/Get-Started/Connect-To-Testnet.md [hunk 1]
```diff
@@ -4,189 +4,182 @@ description: How to choose and connect to a testnet
...
     ```
-    teku --network="https://github.com/goerli/schlesi/raw/master/teku/chain.yaml"
-    ```
-
-**Deposit contract**:
-
-The deposit contract is a deployed instance of the [`validator_registration.vy`](https://github.com/ethereum/eth2.0-specs/blob/dev/deposit_contract/contracts/validator_registration.vy) on an ETH1 network of your choice.
 
-`--eth1-deposit-contract-address="0xaa888248144bc5d584a7f400839d0d912f21c39a"`
+In the command line:
```
Comment (macfarla) [inline docs/HowTo/Get-Started/Connect-To-Testnet.md:96]:
```suggestion
On the command line:
```

File: docs/Reference/CLI/CLI-Subcommands.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ teku validator generate --deposit-amount-gwei=<GWEI>
 teku validator generate --deposit-amount-gwei=32000000000
 ```
 
-Amount to deposit in the Ethereum 1.0 deposit contract.
+Amount to deposit in the Ethereum 1.0 deposit contract. Defaults to 32 ETH.
```
Comment (ajsutton) [inline docs/Reference/CLI/CLI-Subcommands.md:83]: Actually defaults to the minimum ETH required to activate a validator on the specified network.  That's 32ETH for most networks, but only 3.2 when using `--network minimal` config or any network based on it.

File: docs/HowTo/Get-Started/Connect-To-Testnet.md [hunk 3]
```diff
@@ -4,189 +4,182 @@ description: How to choose and connect to a testnet
...
-## Boostrap a new testnet
+    ```bash
+    besu --network=goerli --data-path=./goerli --rpc-http-enabled=true --rpc-http-port=8545 \
+    --rpc-http-api=ETH,NET,WEB3 --sync-mode=FAST --fast-sync-min-peers=2
+   ```
+  
+### Load the deposit account with ETH
 
-If you already have created a custom testnet or plan to do so, Teku allows you to pass different configurations to the beacon-chain node. A custom network requires a couple of things:
+You need an Ethereum 1.0 account that contains at least 32 ETH (plus gas). For the
```
Comment (ajsutton) [inline docs/HowTo/Get-Started/Connect-To-Testnet.md:68]: The amount of ETH required varies based on the network - witti does use 32ETH but the network definition can modify it (minimal uses 3.2ETH).

File: docs/Reference/CLI/CLI-Subcommands.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Subcommands.md:83]: Done

File: docs/HowTo/Get-Started/Connect-To-Testnet.md (hunk 3 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Connect-To-Testnet.md:68]: Done

**Pull request merged**